internal/apiserver/route_post_network_migrate.go

```diff
@@ -0,0 +1,42 @@
+// Copyright © 2022 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package apiserver
+
+import (
+	"net/http"
+
+	"github.com/hyperledger/firefly/internal/coremsgs"
+	"github.com/hyperledger/firefly/internal/oapispec"
+	"github.com/hyperledger/firefly/pkg/core"
+)
+
+var postNetworkMigrate = &oapispec.Route{
+	Name:            "postNetworkMigrate",
+	Path:            "network/migrate",
```

Comment:
I quite liked the "Operator Action" general terminology from the API you described, where this is just one special form of a command and we aren't tying ourselves to this being the only type of command that ever exists.

Illustrating the point, with some starter for 10 spelling suggestions:
What do you think about `network/governance/action`, `network/govaction` or `network/operatoraction`.
Then a `type` under the payload of `migration_v_1_0`.

Reply:
... or I guess is the action being performed here not actually a migration, rather just an `end`/`terminate`/`stop`/`swap`/`jump` type action.

The overall migration is a journey:
1. Deploy a new contract
2. All members add that contract to their config, agreeing who's going to say "go"
4. One member authorized to do so, does this action to signify we're ready to move

Reply:
Still ruminating on the spelling for this API... but I did change the internal terminology to `terminate` over `migrate`, so that will likely bubble up here in some form.

Reply:
For this next pass, I changed the API to `/network/operatoraction` with a `type` of `terminate`.

Still open if sliding in "governance" makes more sense. I'm very slightly concerned about "operator" vs the already-used term "operation" (though not terribly so).

Reply:
Or what about the simpler names `/network/action` and `SubmitNetworkAction`?

Reply:
I like these simpler names 👍 

... notice we still have `operatoraction` in the code - worth changing before we merge?